### PR TITLE
[RHCLOUD-17026] Containerize frontend build process

### PR DESCRIPTION
**Pull request by BlakeHolifield**
* Containerize build process

### CODE REVIEW

File: pr_check.sh [hunk 1]
```diff
@@ -4,11 +4,14 @@
 # Export vars for helper scripts to use
 # --------------------------------------------
 # name of app-sre "application" folder this component lives in; needs to match for quay
-export COMPONENT="insights-inventory" 
-export APP_NAME=`node -e 'console.log(require("./package.json").insights.appname)'`
+export COMPONENT="insights-inventory"
+export APP_NAME=$(node -e 'console.log(require("./package.json").insights.appname)')
 export IMAGE="quay.io/cloudservices/$COMPONENT-frontend"
-export WORKSPACE=${WORKSPACE:-$APP_ROOT}  # if running in jenkins, use the build's workspace
+export IMAGE_TAG=$(git rev-parse --short=7 HEAD)
```
Comment (karelhala) [inline pr_check.sh:10]: Do I understand it correctly that `IMAGE` and `IMAGE_TAG` can be omitted?

Comment (BlakeHolifield) [reply pr_check.sh:10]: The docker image doesn't use them, but we still need them for the quay push. They just don't need to be passed into `docker run`

File: pr_check.sh [hunk 2]
```diff
@@ -4,11 +4,14 @@
 # Export vars for helper scripts to use
 # --------------------------------------------
 # name of app-sre "application" folder this component lives in; needs to match for quay
-export COMPONENT="insights-inventory" 
-export APP_NAME=`node -e 'console.log(require("./package.json").insights.appname)'`
+export COMPONENT="insights-inventory"
+export APP_NAME=$(node -e 'console.log(require("./package.json").insights.appname)')
```
Comment (BlakeHolifield) [inline pr_check.sh:8]: Move into frontend-build.sh

**Review by karelhala — APPROVED** [review]
Looking good! I guess the `frontend-build.sh` file will eventually be removed (once the frontend common builder PR is merged)

**Comment by BlakeHolifield** [comment]
@karelhala Yep, exactly.

**Review by karelhala — APPROVED** [review]
Looking good!

**Comment by karelhala** [comment]
@BlakeHolifield the PR build is failing because of missing image, do you know why?

**Pull request merged**